common/commands/config.go

```diff
@@ -154,17 +154,29 @@ func (cc *ConfigCommand) config() error {
 		if err != nil {
 			return err
 		}
-	} else if cc.details.Url != "" {
-		if fileutils.IsSshUrl(cc.details.Url) {
-			coreutils.SetIfEmpty(&cc.details.ArtifactoryUrl, cc.details.Url)
-		} else {
-			cc.details.Url = clientutils.AddTrailingSlashIfNeeded(cc.details.Url)
-			// Derive JFrog services URLs from platform URL
-			coreutils.SetIfEmpty(&cc.details.ArtifactoryUrl, cc.details.Url+"artifactory/")
-			coreutils.SetIfEmpty(&cc.details.DistributionUrl, cc.details.Url+"distribution/")
-			coreutils.SetIfEmpty(&cc.details.XrayUrl, cc.details.Url+"xray/")
-			coreutils.SetIfEmpty(&cc.details.MissionControlUrl, cc.details.Url+"mc/")
-			coreutils.SetIfEmpty(&cc.details.PipelinesUrl, cc.details.Url+"pipelines/")
+	} else {
+		// non-interactive config
+
+		if cc.details.Url != "" {
+			if fileutils.IsSshUrl(cc.details.Url) {
+				coreutils.SetIfEmpty(&cc.details.ArtifactoryUrl, cc.details.Url)
+			} else {
+				cc.details.Url = clientutils.AddTrailingSlashIfNeeded(cc.details.Url)
+				// Derive JFrog services URLs from platform URL
+				coreutils.SetIfEmpty(&cc.details.ArtifactoryUrl, cc.details.Url+"artifactory/")
+				coreutils.SetIfEmpty(&cc.details.DistributionUrl, cc.details.Url+"distribution/")
+				coreutils.SetIfEmpty(&cc.details.XrayUrl, cc.details.Url+"xray/")
+				coreutils.SetIfEmpty(&cc.details.MissionControlUrl, cc.details.Url+"mc/")
+				coreutils.SetIfEmpty(&cc.details.PipelinesUrl, cc.details.Url+"pipelines/")
+			}
+		}
+
+		// Some build tools support basic authentication only,
+		// to support them we try to extract the username from the access token,
+		// or request the username interactively
```

Comment:
Since we're inbside the "non-interactive config" flow, do you think we should mention the interactive flow here?